ark/utils/spatial_analysis_utils_test.py

```diff
@@ -111,20 +111,21 @@ def test_get_pos_cell_labels_channel():
 
 def test_get_pos_cell_labels_cluster():
     all_data, _ = test_utils._make_dist_exp_mats_spatial_utils_test()
-
+    all_data[settings.CELL_TYPE_NUM] = list(all_data[settings.CELL_TYPE].
+                                            astype('category').cat.codes)
     excluded_channels = [0, 13, 22]
 
     # Subsets the expression matrix to only have channel columns
     channel_start = np.where(all_data.columns == settings.PRE_CHANNEL_COL)[0][0] + 1
     channel_end = np.where(all_data.columns == settings.POST_CHANNEL_COL)[0][0]
 
-    fov_channel_data = all_data.iloc[:, list(range(channel_start, channel_end + 1)) + [31]]
+    fov_channel_data = all_data.iloc[:, list(range(channel_start, channel_end + 1)) + [32]]
```

Comment:
Why do we have this magic number, and why did it change by 1?

Reply:
Great question that also confused me with a lot of failing tests. 
The test_utils function `_make_expression_mat_sa()` that generates an example expression matrix for testing expected the table to have the int cell cluster identifier and made it the 31st column in the data frame. When I deleted that from the list of expected cols, the cell cluster name became column 31 and the int identifiers that were generated and appended in the individual functions became column 32.